src/embed/VbrickEmbedConfig.ts

```diff
@@ -54,6 +55,33 @@ export interface VbrickVideoEmbedConfig extends VbrickBaseEmbedConfig {
 	hideSettings?: boolean;
 	hideFullscreen?: boolean;
 
+	/**
+	 * shows video player with sidebar.
+	 */
+	fullPlayer?: boolean;
+
+	/**
+	 * provide the sidebar name to be opened by default when video loads.
+	 * enableFullVodPlaybackInEmbed value needs to be true for this to work.
+	 */
+	defaultSidebar?: VideoPlaybackSidebarButton;
+
+	/**
+	 * query to filter sidebar items. esp. use in pulse sidebar.
+	 * enableFullVodPlaybackInEmbed = true and defaultSidebar should have a value to make it work.
```

Comment:
does `enableFullVodPlaybackInEmbed` === `fullPlayer` ? I don't see it referenced in the code